wordpress-mobile/WordPress-Android
# Add functionality to upload icon QS task

Fixes #13830 

This PR adds focus point to the site icon. It is smaller than the one on the title and it's in the top left corner of the icon. The size of the focus point is newly set via layout attributes. Once you click on the icon, the focus point and the upload task disappear. You can use the commented out code to test the behaviour.

To test:
- Make sure the "My site improvements" flag is turned on and the app is restarted
- Click on the "Upload site icon" quick start card
- Notice the focus point on the site icon and the snackbar
- Click on the icon
- Notice the focus point disappears
- Notice the QS card disappears

![upload_site_icon](https://user-images.githubusercontent.com/1079756/105374251-c84dcb00-5c07-11eb-86e2-899b2b03fc59.gif)


PR submission checklist:
